core/domain/email_manager.py

```diff
@@ -391,9 +391,6 @@ def send_feedback_message_email(recipient_id, feedback_messages):
         log_new_error('This app cannot send feedback message emails to users.')
         return
 
-    if not feedback_messages:
```

Comment:
I don't think that in normal circumstances this will be needed. Since we are already returning if references are empty. And messages will be empty only if references are empty. So empty references will never make it to `send_feedback_message_email`.
Anyway having a one more if condition will not hurt anything.